cfme/tests/services/test_provision_stack.py

```diff
@@ -395,3 +395,39 @@ def test_stack_template_azure():
         upstream: yes
     """
     pass
+
+
+@pytest.mark.meta(automates=[1698439])
+@pytest.mark.meta(blockers=[BZ(1754543)])
+@pytest.mark.tier(2)
+@pytest.mark.provider([EC2Provider], selector=ONE, scope='module')
+def test_read_dialog_timeout_ec2_stack(order_stack):
+    """
+
+    Bugzilla:
+        1698439
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. create an aws template with an optional value "timeout"
+            2. create a dialog that will offer an option to overwrite "timeout"
+               with a custom value typed at input
```

Comment:
Where is step 1 and 2 happening ?

Reply:
@sshveta from orchestration file we create the template and as per yaml data it gives the dialog input field in order page https://github.com/ManageIQ/integration_tests/tree/master/data/orchestration